packages/core/src/artifacts/OwnedAddressTCRWithAppeals.json

```diff
@@ -1136,8 +1136,8 @@
         "4": {
             "events": {},
             "links": {},
-            "address": "0x79dd3705ba094da565806cc2ceef32c72058dc78",
-            "transactionHash": "0xb99b7bf9ae40b1b27845d30e546227bf7e74bcb5b430a07e35cd47e57ac9a297"
+            "address": "0x81cd72f8f3b49f26177f9a1f36eb727f1e27857a",
+            "transactionHash": "0x6c206be49820cab85e06de87fe86bab59d56f91c014fee8cc295a5156c7ba7b8"
         },
         "50": {
```

Comment:
What about Ganache, did you deploy the new version there too?

Reply:
We're starting to need a better jsons, with versioning. I would love to work on this, especially since 0x has expertize on this, but not enough time for that